### DISCUSSION TITLE
I do not know how to change MODELICAPATH using OMPYthon

### DISCUSSION THREAD

**Discussion by kmerbcn** [OpenModelica/OpenModelica#13160] [Q&A]
Hello

I have a problem building a model using ompython. My script is as follows :

    omc = OMCSessionZMQ();
    
    omc.sendExpression('setTempDirectoryPath("C:/Users/jgarcia/Documents/Pruebas_modelica/tmp/")')
    omc.sendExpression('loadModel(Modelica)');
    omc.sendExpression('loadFile("C:/Users/jgarcia/Documents/Pruebas_modelica/CargadorInalambrico.mo")');
    omc.sendExpression('loadFile("C:/Users/jgarcia/Documents/Pruebas_modelica/Synchronous_Rectifier.mo")');
    omc.sendExpression('loadFile("C:/Users/jgarcia/Documents/Pruebas_modelica/FullBridge.mo")');
    omc.sendExpression('loadFile("C:/Users/jgarcia/Documents/Pruebas_modelica/ResonantePpaerJungCompleto.mo")')
    
    
    os.chdir('C:/Users/jgarcia/Documents/Pruebas_modelica/tmp/');
    omc.sendExpression('cd("C:/Users/jgarcia/Documents/Pruebas_modelica/tmp/")')
    omc.sendExpression('buildModel(ResonantePaperJungCompleto)');

The issue comes as soon as I build the model in the /tmp folder. It seems that I have a problem related the MODELICAPTH.

The error that I can read using getErrorString is as follows  :

 'Error: Failed to load package ResonantePaperJungCompleto (default) using MODELICAPATH C:/Users/jgarcia/AppData/Roaming/.openmodelica/libraries/.\nError: Class ResonantePaperJungCompleto not found in scope <top>.\n'

What should I do?

Thanks in advance for all the people devoting time to answer.

Javier

**Comment by casella** [comment]
@arun3688 can you please help?
